SemiTruck.h

```diff
@@ -1,2 +1,17 @@
 #pragma once
 
+#include <iostream>
+#include <limits>
+#include "Vehicle.h"
+
+struct SemiTruck : public Vehicle
+{
+    SemiTruck(const std::string& n);
+
+
+    
+    void pullInToRoadside();
+    
+    void setSpeed(int s) override;
+};
```

Comment:
Where is your member function to call when the truck is added to the highway?
- [ ] revise